CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - Add multi-invokers; consolidate parameterized-testing documentation
 - Add `fail()` function
 - Remove all test mocks after each test case
+- Add `assert_line_count`
```

Comment:
Can you please move this up to the Unreleased section? The `0.12` was already released https://github.com/TypedDevs/bashunit/releases/tag/0.12.0

Reply:
I tried to sneak this in with my wayback machine. didn't work out it seems ;)